src/main/java/frc/robot/config/Config.java

```diff
@@ -32,6 +36,19 @@
 
     private static final Path ROBOT_ID_LOC = Paths.get(System.getProperty("user.home"), "robot.conf");
 
+    public static FileHandler logFileHandler;
+
+    static {
+        try {
+            String logFilename = new SimpleDateFormat("'Robotlog_'yyyy'-'MM'-'dd'_'HH'-'mm'-'ss'.txt'").format(new Date());
```

Comment:
Cool. We'll need to remember that the logging code is writing files to the roborio file system, and check in every now and then to make sure we're not using up all of its disk space (there isn't much to begin with).

In future we could add logic to auto-delete old log files.